static/scss/partials/landing/a.scss

```diff
@@ -1,9 +1,26 @@
+h1, h2, h3, h4, h5, h6 {
+    font-family: $font-metropolis;
+}
```

Comment:
Can we encapsulate this inside of `.c-landing-hero` (or some other parent class) so it isn't applied globally?

Reply:
Moved it to a parent class for the landing page specifically.